tests/foreman/api/test_contentview.py

```diff
@@ -5,11 +5,15 @@
 from nailgun import client
 from requests.exceptions import HTTPError
 from robottelo import entities
-from robottelo.common.constants import PUPPET_MODULE_NTP_PUPPETLABS
+from robottelo.common.constants import (
+    FAKE_0_PUPPET_REPO,
+    PUPPET_MODULE_NTP_PUPPETLABS
+)
 from robottelo.common.decorators import (
     bz_bug_is_open, data, run_only_on, stubbed)
 from robottelo.common.helpers import get_data_file, get_server_credentials
 from robottelo.test import APITestCase
+import random
```

Comment:
Please, move import statements before any `from ... import`. They should look like the following:

```python
import a
import b
import c

from d import x
from e import y
```